### PR TITLE
reorder and verbose_name fields

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/5388

### CODE REVIEW

**Comment by Pomax** [comment]
@kristinashu I've moved the MSS ding to be the first field, but I've left the spreadsheet showing it as coming after.

File: network-api/networkapi/buyersguide/pagemodels/products/general.py [hunk 1]
```diff
@@ -111,21 +113,22 @@ class GeneralProduct(Product):
...
     ai_is_transparent = ExtendedYesNoField(
-        help_text='Does the company allow users to see how the AI works?',
+        help_text='Does the company allow users to see how the AI works?'
     )
 
     ai_helptext = models.TextField(
         max_length=5000,
         blank=True,
         help_text='Helpful text around AI to show on the product page',
+
```
Comment (mmmavis) [inline network-api/networkapi/buyersguide/pagemodels/products/general.py:131]:
```suggestion
```

**Review by mmmavis — CHANGES_REQUESTED** [review]
found one extra new line but everything else looks good

**Review by mmmavis — APPROVED** [review]
🎉

**Review by kristinashu — APPROVED** [review]
This is great! Ty!

**Pull request merged**